src/Tasks/Microsoft.NET.Build.Tasks/sdk/Sdk.targets

```diff
@@ -49,6 +49,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   <PropertyGroup Condition="'$(NuGetBuildTasksPackTargets)' == '' AND '$(ImportNuGetBuildTasksPackTargetsFromSdk)' != 'false'">
     <IncludeBuildOutput Condition=" '$(PackAsTool)' == 'true' ">false</IncludeBuildOutput>
     <PackageType Condition=" '$(PackAsTool)' == 'true' ">DotnetTool</PackageType>
+    <RuntimeIdentifiers Condition=" '$(PackAsTool)' == 'true' ">$(RuntimeIdentifiers);$(PackAsToolShimRuntimeIdentifiers)</RuntimeIdentifiers>
```

Comment:
I believe this and  the 2 options above it are out of place in the property group. I believe the intention here is that there is a NuGet package for the build tasks that can override the pack targets that come from the SDK. As is, if that is used, then it will break packing tools. I think these should be set unconditionally.

Reply:
cc @rohit21agrawal

Reply:
The whole point of that condition is to allow you to get nuget pack targets out of band from SDK. See #888. 

Suppose the user wants to try a version with a bug fix over what's in the SDK. This should not prevent them from packing a tool with that configuration. 

Also, this file, sdk/sdk.targets (main entry point) is also the wrong place to put code to handle specific scenarios. This should be closer to where PackAsTool is implemented.

Reply:
Exactly as what you said. If I move these props to toolpack, it will fail on muti targeting. Microsoft.NET.Sdk.Common.targets is imported by both. So it is the right place.